server/lib/accounts-manager.js

```diff
@@ -199,14 +199,20 @@ export default class AccountManager {
             operations.push(operation);
         }
 
-        // Create real new operations
+        // Identify the operations to create: an operation can be created
+        // if no duplicate can be found for this operation.
+        let operationsToCreate = [];
         for (let operation of operations) {
             let similarOperations = await Operation.allLike(operation);
             if (similarOperations && similarOperations.length)
                 continue;
+            operationsToCreate.push(operation);
+        }
 
+        // Create the new operations
+        for (let operationToCreate of operationsToCreate) {
             log.info('New operation found!');
```

Comment:
As we now have this information, can you:

`log.info(${operationToCreate.length} new operations found!`

if there are any, please? It's just much better for the logging.